libvcx/src/utils/logger.rs

```diff
@@ -138,12 +138,15 @@ impl LibvcxDefaultLogger {
     pub fn init_testing_logger() {
         trace!("LibvcxDefaultLogger::init_testing_logger >>>");
 
-        // ensures that the test that is calling this wont fail simply because
-        // the user did not set the RUST_LOG env var.
-        let pattern = Some(env::var("RUST_LOG").unwrap_or("trace".to_string()));
-        match LibvcxDefaultLogger::init(pattern) {
-            Ok(()) => (),
-            Err(_) => (),
+        // if RUST_LOG is not set, logs will not be captured
+        match env::var("RUST_LOG") {
+            Ok(log_pattern) => {
+                match LibvcxDefaultLogger::init(Some(log_pattern)) {
+                    Ok(()) => (),
+                    Err(_) => (),
```

Comment:
I think we want to at least log the error message in case the user did set `RUST_LOG` but init failed, no?